.coderabbit.yaml

```
@@ -1,15 +1,70 @@
+  labeling_instructions: []
+  path_filters: []
+  path_instructions: []
+  abort_on_close: true
   auto_review:
     enabled: true
-    drafts: false
+    auto_incremental_review: true
+    ignore_title_keywords: []
+    labels: []
+    drafts: true
+    base_branches: 
```

yamllint

[error] trailing spaces

(trailing-spaces)